packages/components/src/FormField/components/ClearAction.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./ClearAction.css";
+import { Icon } from "../../Icon";
+
+interface ClearActionProps {
+  /**
+   * Click handler
+   */
+  readonly onClick: () => void;
+}
+
+export function ClearAction({ onClick }: ClearActionProps): JSX.Element {
+  return (
+    <div className={styles.clearContainer}>
+      <button
+        className={styles.clearInput}
+        // Prevent the input from losing focus when the clear button is clicked
+        onMouseDown={event => event.preventDefault()}
```

Comment:
I could be misunderstanding - but does this mean that once we mousedown on the clearable, the entire input should still be focused? After I interact with the clearable, no part of the component is in focus anymore.

![Kapture 2023-12-08 at 17 01 00](https://github.com/GetJobber/atlantis/assets/104797704/5b44e7b1-33fa-4a81-9125-9af7a10f70c8)

Reply:
Good catch, I added a focus back to the input on `handleClear`

Reply:
Perfect - works really well now!